src/style.css

```diff
@@ -1,3 +1,72 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+}
+
+html {
+  --primary-blue-color: #00acdc;
+  --header-height: 60px;
+}
+
+.roboto-medium {
+  font-family: Roboto, serif;
```

Comment:
The fallback font for `.roboto-medium` is set to `serif`, which is inconsistent with the `sans-serif` fallback used in `.nav__link`. Consider changing it to `sans-serif` to maintain consistency across your styles.